calico-enterprise/visibility/visualize-traffic.mdx

```diff
@@ -2,7 +2,89 @@
 description: Learn the power of network sets.
 ---
 
-# Visualize traffic to and from a cluster
+# Network Visualization
+> From the left navbar, select **Service Graph**, **Default**
+
+Service Graph offers tools to visualize and investigate network communications in the cluster. To better understand how
+Service Graph works, here is the list of core features and components.
+
+## Service Graph Features and Components
+
+**Namespaces**
+
+Namespaces are the default view in Service Graph.
+
+When you expand the top right panel `<<`, you see a detailed view of the service-to-service communications for the namespace.
+
+![service-graph-namespace](/img/calico-enterprise/service-graph-namespace.png)
+
+**Nodes and edges**
+
+Lines going to/from nodes are called edges. When you click on a node or edge, the right panel shows details, and the associated flow logs are automatically filtered in the bottom panel.
+
+![edges](/img/calico-enterprise/edges.png)
+
+**Layers**
+
+Layers allow you to create meaningful groupings of resources so you can easily hide and show them on the graph. For example, you can group resources for different platform infrastructure types in your cluster like networking, storage, and logging.
+
+> Click the panel on the left (`>>`) by the Namespaces breadcrumb, and then expand the Tigera components layer.
+
+![service-graph-layers](/img/calico-enterprise/service-graph-layers.png)
+
+The **Tigera components** layer contains namespaces for {{prodname}} networking components, and is a view of interest to Dev/Ops.
+
+> Click the vertical ellipses and select, **Hide layer**. Notice that only the business application namespaces remain visible in the graph.
+
+> To make this layer less visible, select **Restore layer** and click **De-emphasize layer**.
+
+**Service groups**
+
+Service group is an abstraction mechanism for grouping services in a single node in the namespace view.
+Services that pass the traffic to the same destination are grouped within a service group.
+
+If services within the service group belong to different namespaces the name of the node in namespace view is printed as "\*",
+otherwise the service group name is printed on the node which is a string combined of all uniques service names.
+
+If you double-click on the service group, you can see details of the services, endpoints, and the network communication
+links (edges) between them. Be aware that services and the backing endpoints could belong to different namespaces.
+
+Here is an examples showing how services and endpoints are grouped under service groups. Note that the example is created
+with synthetic traffic data to showcase the service group feature and do not have any other value.
+
+![a "*" node in namesapce view](/img/calico-enterprise/service-graph/servicegroups.gif)
```

Comment:
This looks odd in the rendered page, the text is not visible.
![image](https://github.com/user-attachments/assets/f42d3c3d-35e7-4ffa-8f81-db68ecab110b)

I think that the sub text should be a full sentence starting with a capital.

Reply:
https://deploy-preview-1600--calico-docs-preview-next.netlify.app/calico-enterprise/next/visibility/visualize-traffic

Reply:
looks like the title shown before the image link is just an internal title and not rendered at all. 
I updated the sentence below the image to be a complete sentence.